build/teamcity/cockroach/ci/tests/acceptance_impl.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+bazel build //pkg/cmd/bazci //pkg/acceptance:acceptance_test //pkg/cmd/cockroach-short:cockroach-short --config=ci
```

Comment:
Also add `--config=crosslinux` -- pretty sure you'll want it because we're shipping that `cockroach-short` to a Docker container

Reply:
Doh! True story!